src/LatteContext/Collector/ComponentCollector.php

```diff
@@ -181,12 +199,7 @@ private function buildComponents(Node $node, Scope $scope, ClassReflection $clas
             }
         }
 
-        $lattePhpDoc = $this->lattePhpDocResolver->resolveForNode($node, $scope);
-        if ($lattePhpDoc->isIgnored()) {
-            return null;
-        }
         if ($lattePhpDoc->hasComponents()) {
-            $components = [];
```

Comment:
other thing I'm not sure with... If I collect some components, and there are some in phpdoc I think I don't want to reset all collected. Just overwrite those which are in php doc and add missing from php doc

Reply:
Correct we did same for variables